packages/functions/queues/process-cron-job-queue.ts

```diff
@@ -0,0 +1,12 @@
+import type { SQSHandler } from "aws-lambda";
+import { Config } from "sst/node/config";
+
+import { sendSlackMessage } from "@/services/slack/sendSlackMessage";
+
+export const handler: SQSHandler = async () => {
+  try {
+    sendSlackMessage(Config.CORE_SLACK_CHANNEL_ID, "Hello from Birthday bot!");
```

Comment:
I guess you should use await here to catch errors.

```suggestion
    await sendSlackMessage(Config.CORE_SLACK_CHANNEL_ID, "Hello from Birthday bot!");
```

Reply:
You're spot on, fixed it.